helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -488,10 +490,10 @@ public boolean scheduleTask(MessageTask task) {
               "Message handling task already sheduled for " + taskId, manager);
         }
       }
-    } catch (Exception e) {
-      LOG.error("Error while executing task. " + message, e);
+    } catch (Throwable t) {
+      LOG.error("Error while executing task. " + message, t);
       _statusUpdateUtil
-          .logError(message, HelixTaskExecutor.class, e, "Error while executing task " + e,
+          .logError(message, HelixTaskExecutor.class, "Error while executing task " + t,
```

Comment:
Please consider not integrate the Throwable in message. If we do it this way, it only print out a short description of this throwable.
We may want to print the stack trace and other information.  Please consider something like. 
``` 
 public void logError(Message message, Class classInfo, Exception e, String additionalInfo,
      HelixManager manager) 
```

Reply:
Can close this one, as created https://github.com/apache/helix/pull/2730 for addressing the comments